index.js

```diff
@@ -41,21 +41,36 @@ const osMap = {
 };
 
 const extMap = {
-    linux: "",
+    linux: ""g
```

Comment:
typo?

Reply:
Yes, was tired :)

Reply:
I updated the commit to fix this and seems to have worked well so far: https://github.com/yrashk/zig-generator/runs/4723582120?

Do you think it is worth merging?